{
	"name": "CVE-2017-18207",
	"seq": "2017-18207",
	"type": "CAN",
	"status": "Candidate",
	"phase": {
		"text": "Assigned",
		"date": "20180228"
	},
	"desc": "** DISPUTED ** The Wave_read._read_fmt_chunk function in Lib/wave.py in Python through 3.6.4 does not ensure a nonzero channel value, which allows attackers to cause a denial of service (divide-by-zero and exception) via a crafted wav format audio file. NOTE: the vendor disputes this issue because Python applications \"need to be prepared to handle a wide variety of exceptions.\"",
	"refs": [
		{
			"text": "https://bugs.python.org/issue32056",
			"source": "MISC",
			"url": "https://bugs.python.org/issue32056"
		},
		{
			"text": "openSUSE-SU-2020:0086",
			"source": "SUSE",
			"url": "http://lists.opensuse.org/opensuse-security-announce/2020-01/msg00040.html"
		}
	],
	"votes": {}
}
